ci/builder/Dockerfile

```diff
@@ -40,7 +40,8 @@ RUN apt-get update --fix-missing && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-ge
     patch \
     texinfo \
     unzip \
-    xz-utils
+    xz-utils \
+    cargo install cargo-vet
```

Comment:
This won't work. See line 188-199 for how to cargo install and use a fixed version for cargo-vet so we don't get any surprises.

Reply:
As soon as I committed I realized it wasn't in the right place. Should be fixed now.